src/modular-etherspot-wallet/modules/validator/MultipleOwnerECDSAValidator.sol

```diff
@@ -1,19 +1,20 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.21;
 
-import "../erc7579-ref-impl/interfaces/IERC7579Account.sol";
-import "../erc7579-ref-impl/interfaces/IERC7579Module.sol";
-import "../erc7579-ref-impl/libs/ModeLib.sol";
-import "../erc7579-ref-impl/libs/ExecutionLib.sol";
-import {ModularEtherspotWallet} from "../wallet/ModularEtherspotWallet.sol";
+import "../../erc7579-ref-impl/interfaces/IERC7579Account.sol";
+import "../../erc7579-ref-impl/interfaces/IERC7579Module.sol";
+import "../../erc7579-ref-impl/libs/ModeLib.sol";
+import "../../erc7579-ref-impl/libs/ExecutionLib.sol";
+import {ModularEtherspotWallet} from "../../wallet/ModularEtherspotWallet.sol";
 import {ECDSA} from "solady/src/utils/ECDSA.sol";
 import {EIP712} from "solady/src/utils/EIP712.sol";
 
 contract MultipleOwnerECDSAValidator is EIP712, IValidator {
     using ExecutionLib for bytes;
     using ECDSA for bytes32;
 
-    bytes32 internal constant EIP712_DOMAIN_TYPEHASH = 0x8b73c3c69bb8fe3d512ecc4cf759cc79239f7b179b0ffacaa9a75d522b39400f;
+    bytes32 internal constant EIP712_DOMAIN_TYPEHASH =
+        0x8b73c3c69bb8fe3d512ecc4cf759cc79239f7b179b0ffacaa9a75d522b39400f;
```

Comment:
What is the domain hash being used here? Please can we add that as a comment.

Reply:
done